exosphere-shared/src/docker-helper.ls

```diff
@@ -84,5 +85,11 @@ class DockerHelper
       child_process.exec-sync 'docker rm -f $(docker ps -aq)'
 
 
+  # Prints the last n number of lines from a given text
+  @_print-last-lines = (text, number-of-lines) ->
+    text
+      .split \\n
```

Comment:
@alexdavid made a good point: since Windows machines split on `\r\n`, if we split on `\n` and then rejoin on `\n` we won't need to use `os.EOL`